docs/ci-cd.md

```diff
@@ -0,0 +1,86 @@
+# Continues integration/delivery
+
+- Unit/Integration tests are executed for each PR
+- Each PR needs to be reviewed
+- PR can be merged only after all tests pass and code is reviewed
+- We use pipeline in code approach and Jenkins as our main CI/CD server
+- CI part of the pipeline (running tests for each PR) is defined in `Jenkinsfile.ci` file. 
+- CI part is run on Hyperledger Jenkins, so it is public and open as every contributor needs to see results of the tests run for his or her PR.
+- CD part of the pipeline (running tests for each PR) is defined in `Jenkinsfile.cd` file.
+- CD part is run on a private Jenkins server dealing with issuing and uploading new builds. 
+
+#### Branches
+
+- Master branch contains the latest changes. All PRs usually need to be sent to master.
+- Stable branch contains latest releases (https://github.com/hyperledger/indy-node/releases). Hotfixes need to be sent to stable.
+
+#### Builds
+
+What artifacts are produced after each push
+- to `master` branch:
+    - indy-plenum:
+        - indy-plenum-dev in [pypi](https://pypi.python.org/pypi/indy-plenum-dev) 
+        - indy-plenum deb package in [https://repo.sovrin.org/deb xenial master-latest](https://repo.sovrin.org/lib/apt/xenial/master-latest/)
+        - indy-plenum release tag (https://github.com/hyperledger/indy-plenum/releases)
+    - indy-node:
```

Comment:
Here I intentionally didn't mention anoncreds, and provide tutorials only for plenum and node, because of the following:
- anoncreds project will be deprecated soon;
- we should ask all people who wants contribution to anoncreds use libindy
- anoncreds project is quite independent of plenum/node (at least more independent than plenum and node pair).